docs/provisioner/08-02-provisioning-gardener.md

```diff
@@ -137,19 +170,26 @@ This tutorial shows how to provision clusters with Kyma Runtimes on Google Cloud
 
   4. Download the service account configuration (`kubeconfig.yaml`) and use it to create a Secret in the `compass-system` Namespace with the key `credentials` and the value encoded with base64.
```

Comment:
We should move this part to requirements, as we expect that kubeconfig is already present during Provisioner installation